src/openms_gui/source/VISUAL/TOPPASEdge.cpp

```diff
@@ -233,7 +233,7 @@ namespace OpenMS
       painter->translate(point);
       painter->rotate(text_angle);
       QFontMetrics fm(painter->fontMetrics());
-      int text_width = fm.width(str);
+      int text_width = fm.width(str); // replace with QFontMetrics::horizontalAdvance if Qt >= 5.11
```

Comment:
should we add a keyword for special todos that relate to minimum versions of contrib libs?
So we can easily search for them in the future, once in a while.